apps/src/storage/firebaseUtils.js

```diff
@@ -94,6 +95,28 @@ export function getProjectDatabase() {
   return getFirebase().child(path);
 }
 
+function getSharedFirebase() {
```

Comment:
Thoughts on refactoring to reduce duplication with `getFirebase`? I'd be tempted to try it and see how it looks, but I could go either way.